### PR TITLE
Fix command pool submitting on shutdown

### PR DESCRIPTION
**Pull request by Owen1212055**
Don't submit to the command pool while on shutdown.

Submitting throws an exception, might just be better to wrap it in a try catch? Unsure.

### CODE REVIEW

**Comment by Owen1212055** [comment]
Now simply has a discard policy, ignoring any submitted tasks that aren't approved.

**Review by electronicboy — APPROVED** [review]

File: patches/server/0296-Async-command-map-building.patch [hunk 1]
```diff
@@ -9,14 +9,15 @@ commands if the server is restarting. Using the default async pool caused issues
...
+index 2bf67468a6c745bc6243c65210477ba129bfcb07..5e390e2e5e3af66cd7b596ecde62c8a29a00ffe6 100644
 --- a/src/main/java/net/minecraft/commands/Commands.java
 +++ b/src/main/java/net/minecraft/commands/Commands.java
-@@ -360,6 +360,23 @@ public class Commands {
+@@ -360,6 +360,25 @@ public class Commands {
          if ( org.spigotmc.SpigotConfig.tabComplete < 0 ) return; // Spigot
          // CraftBukkit start
          // Register Vanilla commands into builtRoot as before
 +        // Paper start - Async command map building
++        // Don't send to pool if it's shutdown
```
Comment (Machine-Maker) [inline patches/server/0296-Async-command-map-building.patch]: I think this comment was left over from before you found out about the discard policies?

Comment (Owen1212055) [reply patches/server/0296-Async-command-map-building.patch]: Yeah see paper contrib, I will change this later.

Comment (Owen1212055) [reply patches/server/0296-Async-command-map-building.patch]: done

**Review by Machine-Maker — APPROVED** [review]

**Review by jpenilla — APPROVED** [review]

**Pull request merged**